lib/resty/core/phase.lua

```diff
@@ -0,0 +1,55 @@
+local ffi = require 'ffi'
+local C = ffi.C
+local base = require "resty.core.base"
```

Comment:
I believe this library consistently groups requires and caching, and separates them by 2 blank lines. It also uses double quotes:
```lua
local ffi = require "ffi"
local base = require "resty.core.base"

local C = ffi.C
local tostring = tostring
local FFI_ERROR = base.FFI_ERROR
```